### PR TITLE
Provide a graphical Windows installer

### PR DESCRIPTION
**Pull request by nezihyigitbasi**
This PR provides a graphical Windows installer for the ODBC driver built using nsis (http://nsis.sourceforge.net). I haven't committed the installer executable with this PR, but you can check it out here: https://github.com/nezihyigitbasi/presto-odbc/tree/binaries/binaries/64-bit. I have successfully tested the installer with Windows 7 and Windows Server 2012.

### CODE REVIEW

File: driver/driver.d [hunk 1]
```diff
@@ -62,7 +62,6 @@ version (Windows) {
         Runtime.initialize();
         import core.memory;
         GC.disable();
-        assert(false, "Debugging an issue whereby this function is not called");
```
Comment (nezihyigitbasi) [inline driver/driver.d:65]: this was failing the os x build as the next line is unreachable

**Comment by nezihyigitbasi** [comment]
Fixes #19 for Windows

File: driver/driver.d (hunk 1 above)
Comment (andralex) [inline driver/driver.d:65]: yah saw that, thanks :)

File: installer/win/installer.nsi [hunk 2]
```diff
@@ -0,0 +1,99 @@
+!define APPNAME "Presto ODBC Driver"
```
Comment (andralex) [inline installer/win/installer.nsi]: Should you add "64 bit" somewhere in the name as well?

File: Makefile [hunk 3]
```diff
@@ -3,23 +3,34 @@ LIBCURL = /cygdrive/c/D/dmd2/windows/bin64/libcurl.dll
...
 
+MAKE_NSISW_CMD = "/cygdrive/c/Program Files (x86)/NSIS/makensis.exe"
+MAKE_NSIS_FLAGS = /V4
+
 DC = LINKCMD64="$(VS_HOME)\bin\link.exe" dmd
 CFLAGS = -c
 FLAGS = -g -w -version=UNICODE
+
 ifeq ($(OS),Windows_NT)
+	#Installer
```
Comment (andralex) [inline Makefile]: Some comments here are rather unnecessary - "#Installer" precedes INSTALLER_XXX stuff, "#Flags" precedes FLAGS etc. May be removed without any harm.

**Comment by andralex** [comment]
LGTM, a few nits only - please let us know if any change and then let's pull this.

**Comment by nezihyigitbasi** [comment]
@andralex Addressed your comments, thanks for looking into this.

**Comment by andralex** [comment]
merged, thanks much

**Pull request closed without merging**